app/views/buyers/service_contracts/bulk/shared/errors.html.erb

```diff
@@ -1,7 +1,7 @@
 <div class="bulk_operation errors">
   <h1>There were some errors:</h1>
-  <%- @errors.each do |application| -%>
-    <h2 class="contracts"><strong><%= contract.name %></strong> (<%= contract.account.org_name %>)</h2>
+  <%- @errors.each do |contract| -%>
+    <h2 class="contracts"><strong><%= contract.service.name %></strong> (<%= contract.account.org_name %>)</h2>
```

Comment:
Maybe the text could be something like 
```suggestion
    <h2 class="contracts">Subscription of <%= contract.account.org_name %> to service <%= contract.service.name %></h2>
```

In any case, looks like we don't need `<strong>` because `<h2>` already be "strong".

Reply:
The error template is so terribly ugly I wouldn't even spend time on this 😂 I mean...
![Screenshot 2023-10-06 at 17 21 41](https://github.com/3scale/porta/assets/11672286/018cdaee-e870-4deb-ab5a-e2ea46a5d646)


Inspired by your suggestion I came up with the following:
![Screenshot 2023-10-06 at 17 21 16](https://github.com/3scale/porta/assets/11672286/8251838f-6865-47f7-9a12-20efabf122fa)

Reply:
👍🏻 Done